### PR TITLE
Plasma cutters are more expensive

### PR DESCRIPTION
**Pull request by MacHac**
### Intent of your Pull Request
Plasma cutters are hilariously cheap for how good they are.  If you have it in you to haul one plasma ore back to the station, your prize is a mining tool that blows every comparable tool out of the water.  For that plus a sheet of gold, you get an even better version of the same thing.

I don't fault plasma cutters their power since PKA mining is about as entertaining as banging your head against a wall.  However, there's no real point to mining before science gets mining tech since regular mining is so slow and inefficient.

So, here's the deal.  You want your shiny plasmacutter?  You have to find at least one uranium deposit the old fashioned way.  You want it advanced?  Either get lucky and find some diamonds or kill a watcher.

#### Why good 4 game
First, it makes sure the station at least has one sheet of everything before the miners run off to die to Bubblegum.  Second, it rewards people who mine before science researches advanced cutters.  Third, it might give some actual reason to use regular cutters besides science sucking out loud.  Fourth, it encourages people to take risks they might not usually take, like fighting a watcher rather than running straight past.

#### Changelog

:cl:  
tweak: Plasma cutters are more expensive.
/:cl:

_Disclaimer: numbers completely made up on a whim, feel free to suggest changes._

### CODE REVIEW

**Comment by ynot01** [comment]
for reference

**plasma cutter old**
1 metal sheet
1 glass sheet
1 plasma sheet
**plasma cutter new**
1 metal sheet
1 glass sheet
1 plasma sheet
1 silver sheet
1 uranium sheet

**adv pc old**
2 metal sheets
1 glass sheet
1 plasma sheet
1 gold sheet
**adv pc new**
2 metal sheets
1 glass sheet
1 plasma sheet
1 gold sheet
2 uranium sheets
1 diamond sheet

**Comment by Takahiru** [comment]
will literally change nothing because i just use crusher and kpa anyways

**Comment by boodaliboo** [comment]
ki eat hgaMBURTG

**Comment by ynot01** [comment]
i feel like the word "encourages" is misleading, making it sound like a good thing when this just makes it more tedious to get the tools that make life bearable

you even said yourself that PKA mining is like banging your head against the wall, why would you extend that boring gameplay

its not like miners (most of them anyways) AFK in cargo until the other miners haul up the ores so they can print plasma cutters

all this is going to do is have people rush the newly required mats and run back for a plasma cutter

and any miners with more than 2 brain cells arent going to run off into megafauna headfirst before they collect mats because in order to fight that megafauna they need to print upgrades which cost them even more mats

all in all this PR is just going to slow down miner progression, not provide more mats for the station overall

**Comment by MacHac** [comment]
If miners win because they do a good job, they'll do a good job more often.  The only things to bring miners back to the station are equipment and points; miners can only buy so many survival pens before their backpack is full, so offering good equipment is the best way to incentivize miners to bring high-level materials back.

Right now cutters are so comically cheap that there's no real incentive to maximize ore return.  If the station doesn't have any diamond, mining doesn't suffer; the adv cutter is the ONLY piece of top-tier mining equipment that doesn't require diamond to make.  Likewise, one deposit of uranium will see the miners set for the round while starving the rest of the station.

Jacking up equipment costs incentivizes miners to seek uncommon materials even at cost to themselves, such as time spent fighting watchers instead of killing goliaths for plates.  It also encourages miners to move quickly if they want to maximize their hunting time, which adds an extra element of challenge to the job.

Bottom line: PKA mining is boring in large doses, but running around trying to find a diamond before the first storm adds an extra layer of difficulty for the robust without making lavaland harder on inexperienced miners.  The real goal is to make sure the station has advanced materials even if the miners are gamers that halfass the job to go kill fauna.

**Comment by ynot01** [comment]
if you want miners to bring back more mats the solution isnt to increase lathe prices which will never matter because everything will always cost 2 sheets of something max

increase mining vendor prices because that shit actually costs a lot

target things that miners want like a hardsuit or survival pens and shit
then they'll mine more so they can get more points

**Comment by boodaliboo** [comment]
who invented mine

**Comment by Hopekz** [comment]
Letting this marinate a little bit longer for more opinions.
At the moment it looks as if this might be a merge.

**Review by Hopekz — APPROVED** [review]
After reviewing all the arguments made by everyone I think we can give this a try to see how it affects things.

**Comment by deathslayer2481** [comment]
This is kind of dumb, and will just result in mining being slower. Instead of increasing cost of PC, increase the effectiveness of the KA for mining purposes. On a side note, also make resonator and crusher better mining tools.
